components/script/dom/webidls/XRInputSource.webidl

```diff
@@ -24,4 +24,7 @@ interface XRInputSource {
   [SameObject] readonly attribute XRSpace? gripSpace;
   // [SameObject] readonly attribute Gamepad? gamepad;
   /* [SameObject] */ readonly attribute /* FrozenArray<DOMString> */ any profiles;
+
+  [Pref="dom.webxr.hand"]
+  readonly attribute XRHand? hand;
```

Comment:
I guess this is a spec nit, but `hand` singular?

Reply:
An input source only has one hand. If your input source has multiple hands, it might be time to call your doctor and/or plastic surgeon.

Reply:
![image](https://user-images.githubusercontent.com/1617736/80390805-c8c85980-8861-11ea-9410-a17676a7c005.png)

Reply:
(This also matches with gamepad -- a single input source is a single controller in a single hand, and it can have one gamepad and one associated hand)